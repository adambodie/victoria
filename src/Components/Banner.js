import React from 'react';

const Banner = props => <div><img src={require(`../Images/${props.name}.jpg`)} alt={props.name} className='img-fluid'/></div>;

export default Banner;
