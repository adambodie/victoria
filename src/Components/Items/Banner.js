import React from 'react';

const Banner = props => <div><img src={`https://s3-us-west-2.amazonaws.com/victoria.bodiewebdesign.com/Images/${props.name}.jpg`} alt={props.name} className='img-fluid'/></div>;

export default Banner;
