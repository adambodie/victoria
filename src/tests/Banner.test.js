import React from 'react';
import renderer from 'react-test-renderer';
import { Link } from 'react-router-dom';
import { MemoryRouter } from 'react-router';
import Banner from '../Components/Items/Banner';

it('renders a banner component', () => {
  const tree = renderer.create(<Banner name="Name" alt="Alt" />).toJSON();
  expect(tree).toMatchSnapshot();
});
