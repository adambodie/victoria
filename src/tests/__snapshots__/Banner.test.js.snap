// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a banner component 1`] = `
<div>
  <img
    alt="Alt"
    className="img-fluid"
    src="https://victoria.bodiewebdesign.com/Images/Name.jpg"
  />
</div>
`;
